chain/arweave/src/chain.rs

```diff
@@ -1,3 +1,5 @@
+// Portions copyright (2023) Vulcanize, Inc.
+
```

Comment:
Can we please remove these copyright notices? That goes for all the copyright notices that this PR introduces.

Reply:
Somehow github showed me an old version of the file.